packages/e2e-tests/src/config/bootstrap.js

```diff
@@ -112,6 +112,11 @@ const pageEvents = [];
 // The Jest timeout is increased because these tests are a bit slow
 jest.setTimeout(PUPPETEER_TIMEOUT || 100000);
 
+// Retry flaky tests at most 2 times in CI (off by 1).
```

Comment:
I don't think this is correct though. I believe `retryTimes(3)` means it will at most run the test 4 times, 1 normal run, then 3 retries.